working_with_fhir/working_with_fhir_parsing.bal

```diff
@@ -5,14 +5,14 @@ import ballerina/io;
 public function main() returns error? {
 
     // The following example is a simple serialized Patient resource to parse
-    string input = "{" +
-    "\"resourceType\" : \"Patient\"," +
-    "  \"name\" : [{" +
-    "    \"family\": \"Simpson\"" +
-    "  }]" +
-    "}";
+    json input = {
+        "resourceType" : "Patient",
+        "name" : [{
+            "family": "Simpson"
+        }]
+    };
 
-    // Parse it - you can pass the input string or a json as input and the 
+    // Parse it - you can pass the input (as a string or a json) and the
     // type of the resource you want to parse.
     fhir:Patient patient = check fhirParser:parse(input, fhir:Patient).ensureType();
```

Comment:
```suggestion
    fhir:Patient patient = check input.cloneWithType();
```
Or we can completely replace the statement with language built in functions

Reply:
It's not a clone?! can you elaborate how this works? we are parsing fhir using a ballerinax lib here.

Reply:
@nirmal070125  It is a clone with a given type.  you can use like this too : `fhir:Patient patient = check input.cloneWithType(fhir:Patient);` But the type as an argument, is not needed as we have the type in the LHS, so it is inferred. Inside the `parse` function the `clone` function is called anyways, because you are creating a new type. 

The reason to suggest this change is as we have a tag line saying "FHIR? Ballerina is FHIR", I thought it would be better to use Ballerina inbuilt features. :)